src/content_mapblock.cpp

```diff
@@ -30,22 +30,36 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 #include "noise.h"
 
+#define SMOOTH_LIGHTING_OVERSIZE 0.0f
+
+typedef u16 LightFrame[8];
```

Comment:
It is intended to be something opaque, used only to transfer some data from `getSmoothLightFrame` to `blendLight`. E.g. it could contain the node’s own light level, or its `light_source` value (it would be a structure then), etc.
Surely I can change that, but only is you consider that necessary.